examples/fluids/qfunctions/stabilization.h

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) 2017-2022, Lawrence Livermore National Security, LLC and other CEED contributors.
+// All Rights Reserved. See the top-level LICENSE and NOTICE files for details.
+//
+// SPDX-License-Identifier: BSD-2-Clause
+//
+// This file is part of CEED:  http://github.com/ceed
+
+/// @file
+/// Helper functions for computing stabilization terms of a newtonian simulation
+
+
+#ifndef stabilization_h
+#define stabilization_h
+
```

Comment:
```suggestion
#include "newtonian_state.h"
#include <ceed.h>
```

Stop some unnecessary warnings from my LSP.